phocoa/framework/WFAuthorization.php

```diff
@@ -535,11 +535,15 @@ function login($username, $password, $passIsToken = false)
      *
      * @param object WFAuthorizationInfo The new authorization info to set for the current session.
      */
-    function loginAsAuthorizationInfo($authInfo, $authorizeRecentLogin = true)
+    function loginAsAuthorizationInfo($authInfo, $authorizeRecentLogin = true, $temporary = false)
     {
         if (!$authInfo instanceof WFAuthorizationInfo) throw new WFException("WFAuthorizationInfo or subclass required.");
 
         $this->authorizationInfo = $authInfo;
+        
+        if($temporary){
+      			return;
+      		}
```

Comment:
@parisholley not sure what to make of this... does this still need reviewing? w/o some docs/context I am not sure how to eval.